test/test_mount.rs

```diff
@@ -207,15 +207,22 @@ macro_rules! run_tests {
 
 #[cfg(target_os = "linux")]
 fn main() {
-    use test_mount::{setup_namespaces, test_mount_tmpfs_without_flags_allows_rwx,
-                     test_mount_rdonly_disallows_write, test_mount_noexec_disallows_exec,
-                     test_mount_bind};
-    setup_namespaces();
-
-    run_tests!(test_mount_tmpfs_without_flags_allows_rwx,
-               test_mount_rdonly_disallows_write,
-               test_mount_noexec_disallows_exec,
-               test_mount_bind);
+    use std::env;
+
+    if env::var("DOCKER_ENVIRONMENT").is_ok() {
```

Comment:
actually probably better is to change the panic in `setup_namespaces` to just exit after printing a warning. not having unprivileged namespaces available is not a test failure. thoughts?